src/main/scala/modbat/cov/Trie.scala

```diff
@@ -338,14 +338,14 @@ class TransitionInfo(val transOrigin: State,
                      val transitionID: Int,
                      var transCounter: Int,
                      val transitionQuality: Quality,
-                     val nextStateNextIf: Transition#NextStateNextIf,
+                     val transToNextState: Transition#transToNextState,
```

Comment:
The part after "#" is an inner class name; it should be kept as is, otherwise the code won't compile.